src/lib/support/jsontlv/BUILD.gn

```diff
@@ -28,6 +28,7 @@ static_library("jsontlv") {
 
   public = [
     "JsonToTlv.h",
+    "JsonUtilities.h",
```

Comment:
I would avoid names of utilities/helpers/common/support and other such fillers that do not convey the intent (and as a result they end up being a grab all of various mixed unrelated things).

In this case maybe "TextFormat.h" could explain both prettyprint and singleline.

Reply:
done, thanks